src/applications/static-pages/download-1095b/components/App/index.js

```diff
@@ -214,12 +203,12 @@ export const App = ({ displayToggle, isLOA1, loggedIn, toggleLoginModal }) => {
     </VaAlertSignIn>
   );
 
-  if (!displayToggle) {
-    return unavailableComponent();
-  }
   if (loggedIn) {
-    if (formError.error) {
-      return getErrorComponent();
+    if (!displayToggle) {
+      return unavailableComponent();
+    }
```

Comment:
Moving this in here fixes a issue where the sign-in alert would flicker into view before being replaced.